backend/app/services/launch_pad.rb

```diff
@@ -7,26 +7,17 @@ def initialize(study_id:, user_id:)
     @user_id = user_id
   end
 
-  def launch
+  def launch_url(preview: false)
     raise(LaunchError, 'This study is not open.') unless study.open?
-
-    url = ActiveRecord::Base.transaction do
-      raise(LaunchError, 'You have already completed this study.') if launched_study.completed?
-
-      launch = user.launch_next_stage!(study)
-      launch_config = launch.stage.config
-      launcher =
-        case launch_config[:type]
-        when 'qualtrics'
-          QualtricsLauncher
-        else
-          raise "Unsupported stage type: '#{launch_config[:type]}'"
-        end
-
-      launcher.new(config: launch_config, user_id: user_id).url
-    end
-
-    url || raise('An error occurred when building a launch url')
+    if preview
+      study.stages.order(:order).first.launcher(user_id).url + '&Q_CHL=preview&Q_SurveyVersionID=current'
```

Comment:
Hmm, this is a Qualtrix-specific method of making a preview URL - knowledge of qualtrix-ness is buried in the launcher, so I think we should extend that model to have a preview_url component, instead?

Reply:
good catch, yep this should live in qualtrics launcher.  other launchers might not support preview and could raise an exception or something